sdk/core/azure-core-experimental/dev_requirements.txt

```diff
@@ -1,5 +1,7 @@
 -e ../../../tools/azure-sdk-tools
 ../azure-core
+-e ../azure-core/tests/testserver_tests/coretestserver
 js
 pyodide==0.19.0a1; python_version >= '3.8'
 httpx==0.23.3
+urllib3>=2.0.0
```

Comment:
Why is this a dev requirement only?

Reply:
The experimental package doesn't take a direct dependency on any of the experimental transport packages. Though we could add them as optional dependencies.